product_extended_segmentation/model/template.py

```diff
@@ -216,15 +215,20 @@ def _factor(factor, product_efficiency, product_rounding):
         if float_is_zero(diff, precision_id) or \
                 (current_price and diff < 0 and
                     computed_th > bottom_price_threshold):
-            tmpl_obj.message_post(cr, uid, [product_tmpl_id.id],
+            product_ids = prod_obj.search(
+                cr, uid, [('product_tmpl_id', '=', product_tmpl_id.id)])
```

Comment:
I do not understand call me please.

This is being executed in the context of a product, why a search?